Dockerfile

```diff
@@ -167,9 +167,6 @@ ARG ORCHESTRATOR_URL
 ARG LASSIE_EVENT_RECORDER_AUTH
 ARG LASSIE_EVENT_RECORDER_URL
 
-# Use random peerId until this is fixed https://github.com/filecoin-project/lassie/issues/191
-ARG LASSIE_EXCLUDE_PROVIDERS="QmcCtpf7ERQWyvDT8RMYWCMjzE74b7HscB3F8gDp5d5yS6"
```

Comment:
I think we still need the variable declaration `ARG LASSIE_EXCLUDE_PROVIDERS`, because it's referenced below. We can still remove the default value though.

Reply:
Is this no longer used? Or what is the reason to remove it?

Reply:
Than can we just also remove L186? I'm fine with keeping this. I was following the comment on L170. The bug mentioned in the comment was fixed, so I was going to remove it.